.circleci/config.yml

```diff
@@ -0,0 +1,282 @@
+                          --install --wait \
+                          --namespace=laa-maat-orchestration-<< parameters.environment >> \
+                          --values ./helm_deploy/laa-maat-orchestration/values-<< parameters.environment >>.yaml \
+                          --set image.tag="${CIRCLE_SHA1}"
+
+# ------------------
+# JOBS
+# ------------------
+jobs:
+  build_and_scan:
+    executor: build-executor
+    steps:
+      - checkout:
+          path: ~/laa-maat-orchestration
+      - restore_cache:
+          key: v1-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - restore_cache:
+          key: v1-gradle-cache-{{ checksum "build.gradle" }}
+      - run:
+          name: Build
+          command: ./gradlew clean build
+      - run:
+          name: Run SonarQube
+          command: ./gradlew sonarqube -Dsonar.projectKey=${SONAR_PROJECT_KEY} -Dsonar.host.url=${SONARQUBE_URL} -Dsonar.organization=${SONAR_ORG}
+      - save_cache:
+          paths:
+            - ~/.gradle/wrapper
+          key: v1-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - save_cache:
+          paths:
+            - ~/.gradle/caches
+          key: v1-gradle-cache-{{ checksum "build.gradle" }}
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+      - store_test_results:
+          path: build/test-results/test
+      - store_artifacts:
+          path: build/test-results/test
+      - store_artifacts:
+          path: build/reports/jacoco/test/html
+      - store_artifacts:
+          path: build/libs
+      - snyk/scan:
+          <<: *snyk_options
+
+  build_scan_and_push_image:
+    executor: build-executor
+    steps:
+      - checkout:
+          path: ~/laa-maat-orchestration
+      - attach_workspace:
+          at: .
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - aws-cli/setup:
+          role_arn: $ECR_ROLE_TO_ASSUME
+          region: $ECR_REGION
+      - run: |
+          aws ecr get-login-password --region $ECR_REGION | docker login --username AWS --password-stdin ${AWS_ECR_REGISTRY_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com
+      - run:
+          name: Build docker image
+          command: |
+            docker build \
+            --build-arg COMMIT_ID=${CIRCLE_SHA1} \
+            --build-arg BUILD_TAG=${CIRCLE_SHA1} \
+            --build-arg APP_BRANCH=${CIRCLE_BRANCH} \
+            --build-arg BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z) \
+            -t $AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1} .
+      - snyk/scan:
+          docker-image-name: "$AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1}"
+          <<: *snyk_options
+      - run:
+          name: Push to ECR
+          command: |
+            docker push "$AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/laa-maat-orchestration-dev-ecr:${CIRCLE_SHA1}"
```

Comment:
Please chekc this, it should be $AWS_ECR_ACCOUNT_URL/laa-crime-apps-team/**laa-maat-orchestration-dev**

Reply:
ear repo name created by terraform has ecr appended to the namespace `repo_name = "${var.namespace}-ecr"` I updated the helm values file accordingly.

Reply:
agree. The repo name in the terraform for ecr doesn't have -ecr in the latest update.